Binaries/bin-conf/flhook_plugins/laz_laws.cfg

```diff
@@ -0,0 +1,311 @@
+;; If enabled, the /laws command will work.
+[LawsConfig]
+Enabled = yes
+
+;; If enabled, people with be notified upon jumping into a system when their cargo is illegal in said system.
+[ContrabandConfig]
+Enabled = yes
+
+[Houses]
+Liberty
+Rheinland
+Kusari
+Gallia
+Bretonia
+The Republic of Malta
+Omicron Rho;The Core
+Omicron Mu;The Order
+The Corsair Empire
+The Republic of Crayter
+
+[Omicron Mu]
+system = st08
+contraband = commodity_order
+contraband = commodity_ga_artifacts
+contraband = commodity_xeno_relics
+contraband = commodity_slaves
+contraband = commodity_cardamine
+contraband = commodity_alien_artifacts
+contraband = commodity_corehardware
+contraband = commodity_liquid_cardamine
+contraband = commodity_cardamine_rt
+contraband = commodity_nomad
+contraband = commodity_minorbrain
+contraband = commodity_majorbrain
+contraband = commodity_gatecell
+contraband = commodity_gatecell_bw12
+contraband = commodity_humanorgans
+
+[Omicron Rho]
+system = bw71; wtf?
```

Comment:
wtf?

Reply:
Forgot to remove it. Added it when I sent to be tested by someone else, explained why we have such silly numbers.